crawl-ref/source/util/tag-35-upgrade.py

```diff
@@ -0,0 +1,387 @@
+            "i-gain-dexterity POT_REMOVEME3\n",
+            "i-gain-intelligence POT_REMOVEME4\n",
+            "# Remove POT_POISON and POT_SLOWING when TAG_MAJOR_VERSION > 34\n",
+            "i-poison POT_POISON\n",
+            "i-slowing POT_SLOWING\n",
+            "# Remove porridge when TAG_MAJOR_VERSION > 34\n",
+            "i-porridge POT_PORRIDGE\n",
+            "# please remove these 2 when TAG_MAJOR_VERSION is bumped\n",
+            "i-decay POT_REMOVEME5\n",
+            "i-confusion POT_REMOVE_ME\n",
+            "# remove these 2 when TAG_MAJOR_VERSION is bumped\n",
+            "i-restore-abilities POT_REMOVEME6\n",
+            "i-strong-poison POT_REMOVEME7\n",
+            "# please remove POT_CURE_MUTATION when TAG_MAJOR_VERSION is bumped\n",
+            "i-cure-mutation POT_REMOVEME8\n",
+            "# Remove all blood blood when TAG_MAJOR_VERSION > 34\n",
+            "i-blood POT_BLOOD\n",
+            "i-coagulated-blood POT_BLOOD_COAGULATED\n",
+            "# Remove beneficial mutation when TAG_MAJOR_VERSION > 34\n",
+            "i-good-mutation POT_REMOVEME9\n",
+            "# please remove the following placeholder potion when TAG_MAJOR_VERSION > 34\n",
+            "# (corresponds to POT_DUMMY_AGILITY)\n",
+            "i-stabbing POT_REMOVEME10 POT_ID_LAST\n",
+            "# Remove power when TAG_MAJOR_VERSION > 34\n",
+            "i-staff_power STAFF_POWER\n",
+            "# Remove when TAG_MAJOR_VERSION > 34\n",
+            "i-staff_enchantment STAFF_ENCHANTMENT\n",
+            "# Remove channeling and update STAFF_ID_LAST when TAG_MAJOR_VERSION > 34\n",
+            "i-staff_channeling STAFF_CHANNELING STAFF_ID_LAST\n",
+            "# Remove rods TAG_MAJOR_VERSION > 34\n",
+            "%sdir item/rod\n",
+            "rod00 ROD\n",
+            "rod01\n",
+            "rod02\n",
+            "rod03\n",
+            "rod04\n",
+            "rod05\n",
+            "rod06\n",
+            "rod07\n",
+            "rod08\n",
+            "rod09\n",
+            "# Remove bottle when TAG_MAJOR_VERSION > 34\n",
+            "misc_bottle MISC_BOTTLED_EFREET\n",
+            "# Remove crystal ball when TAG_MAJOR_VERSION > 34\n",
+            "misc_crystal MISC_CRYSTAL_BALL_OF_ENERGY UNSEEN_MISC\n",
+            "# Remove fan when TAG_MAJOR_VERSION > 34\n",
+            "misc_fan MISC_FAN_OF_GALES\n",
+            "misc_fan_inert MISC_FAN_OF_GALES_INERT\n",
+            "# Remove lamp when TAG_MAJOR_VERSION > 34\n",
+            "misc_lamp MISC_LAMP_OF_FIRE\n",
+            "misc_lamp_inert MISC_LAMP_OF_FIRE_INERT\n",
+            "# Remove stone when TAG_MAJOR_VERSION > 34\n",
+            "misc_stone MISC_STONE_OF_TREMORS\n",
+            "misc_stone_inert MISC_STONE_OF_TREMORS_INERT\n",
+            "# Remove when TAG_MAJOR_VERSION > 34\n",
+            "misc_lantern MISC_LANTERN_OF_SHADOWS\n",
+            "# Remove when TAG_MAJOR_VERSION > 34\n",
+            "misc_sack MISC_SACK_OF_SPIDERS\n",
```

Comment:
I think the better approach might be to remove everything between `# REMOVE WHEN TAG_MAJOR_VERSION > 34 {` and `# }` line comments. Same for everything else that uses # comments, e.g. des files. These files don't support any sort of ifdef mechanism, so you shouldn't need to actually evaluate the condition; a textual match should suffice

Reply:
That makes sense, but I'd rather not rewrite this all now unless you think it's really important :\

Reply:
I did it! The comment format is standardised now